oughta/README.md

```diff
@@ -48,7 +51,7 @@ module Main (main) where
 
 import Control.Monad (filterM, forM)
 import Data.ByteString qualified as BS
-import FileCheck qualified as FC
+import Oughta qualified as Ota
```

Comment:
`Oughta` is short enough where I wonder if we should just use `Oughta.foo` instead of `Ota.foo`.